### PR TITLE
Milestone 3 - Read

### PR DESCRIPTION
**Pull request by michaelpfohl**
## Description
Allows the user to add a new item and see that item added immediately to the DOM.

## Related Issue
- Milestone 3 - Read

## Motivation and Context
Lets the user visualize the data.

## How Can This Be Tested?
[Firebase Deploy](https://jurassic-world-eb567.web.app)

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/javascripts/helpers/data/staffData.js [hunk 1]
```diff
@@ -3,10 +3,24 @@ import apiKeys from '../apiKeys.json';
 
 const baseUrl = apiKeys.firebaseKeys.databaseURL;
 
-const addStaff = (staffData) => axios.post(`${baseUrl}/staff.json`, staffData)
-  .then((response) => {
+const addStaff = (staffData) => new Promise((resolve, reject) => {
```
Comment (drteresavasquez) [inline src/javascripts/helpers/data/staffData.js:6]: 💪🏽🎉

**Review by drteresavasquez — CHANGES_REQUESTED** [review]
Excellent. The only thing that I am requesting is that you have a default view on application load. Currently the screen is blank.

File: src/javascripts/helpers/data/staffData.js (hunk 1 above)
Comment (danapham) [reply src/javascripts/helpers/data/staffData.js:6]: this is done!

**Review by drteresavasquez — APPROVED** [review]
Approved!

**Pull request merged**